ddtrace/bootstrap/sitecustomize.py

```diff
@@ -10,12 +10,20 @@
 
 from ddtrace.utils.formats import asbool, get_env
 
+logs_injection = asbool(get_env('logs', 'injection'))
+DD_LOG_FORMAT = '%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s'.format(
+    '[dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] ' if logs_injection else ''
+)
 
 debug = os.environ.get("DATADOG_TRACE_DEBUG")
+
+# Set here a default logging format for basicConfig since we would not be able
+# to change format later without jumping through hoops
+# See https://github.com/python/cpython/blob/2.7/Lib/logging/__init__.py#L1550
```

Comment:
When linking to something in GitHub on the page hit `y` and it'll update the url to link to the latest commit sha, that way this link will always link to what you expected... just in case they decide to make changes to this file.

Interesting, I wasn't expecting the code for `2.7` to be that different from `3.7`, but it is.

Reply:
Yep I didn't expect either! The addition of style and handler arguments is why.